### PR TITLE
Added Norwegian-Bokmål translation

### PR DESCRIPTION
**Pull request by Codivorous**

### CODE REVIEW

**Comment by m2049r** [comment]
Thank you!
Strings marked with ```translatable="false"``` should not be translated (i.e. should be left out of the translated file). @erciccione maybe you can add this to the taiga instructions.
Please rebase to current master as we have new texts in ```strings.xml``` and a section in ```help.xml```.

**Comment by Codivorous** [comment]
Oh, okay! I thought they should be left in, but not translated. I'll rebase and push :)

**Comment by erciccione** [comment]
@m2049r actually is already in the instructions, maybe is not clear enough, will check it and rewrite if necessary

**Comment by Codivorous** [comment]
I believe I got everything :) Also, the language codes for Norwegian are confusing. Some projects use "nb-no" and "nn-no" for bokmål and nynorsk, some only have one (aka. "no") and some have "nb" and "nn". I don't know which one to use, so I just used nb.

**Review by erciccione — APPROVED** [review]
Looks goo to me, thanks @Codivorous Can you squahs those commits in just one?

**Comment by Codivorous** [comment]
I don't have access to a Git client at the moment unfortunately, as I've just done this through the web interface. Don't know how I would do that, sorry :p

**Comment by Codivorous** [comment]
I'm fine with you forking my thing, squashing it and then pushing it btw. I don't care about the credits :p

**Comment by erciccione** [comment]
Don't worry it's not a big deal, maybe @m2049r can just use the "squash and merge" option here on github

**Comment by Codivorous** [comment]
Oh, okay! ^_^

**Comment by erciccione** [comment]
Please revert the last commit (a175fdb) those strings don't have to be translated by they must be there

**Comment by Codivorous** [comment]
>Strings marked with translatable="false" should not be translated (i.e. should be left out of the translated file).

**Comment by erciccione** [comment]
@Codivorous I'm definitely still feeling the friday night, ignore my previous comment.
@m2049r now i got it. updating the instructions

File: app/src/main/res/values-nb/strings.xml [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+    <string name="info_xmrto"><![CDATA[
+        <b>Du skrev inn en Bitcoin addresse.</b><br/>
+        <i>Du vil sende XMR og mottakeren vil få BTC gjennom <b>XMR.TO</b> tjenesten.</i>
+    ]]></string>
+
+    <string name="info_send_xmrto_success_order_label">XMR.TO Ordre</string>
+
+    <string name="info_send_xmrto_success_sending">Sender</string>
+    <string name="info_send_xmrto_success_status_label">Status:</string>
```
Comment (m2049r) [inline app/src/main/res/values-nb/strings.xml]: pls delete this (info_send_xmrto_success_status_label) - does not exist in original

Comment (m2049r) [inline app/src/main/res/values-nb/strings.xml]: please delete this (info_send_xmrto_success_status_label) - does not exist in original

File: app/src/main/res/values-nb/help.xml [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+		uten å være redd for å miste penger. Det er denne folk sender Monero til.
+        <h2>Mnemonisk Seed</h2>
+        Dette er den eneste dataen du trenger for å gjenoprette lommeboka di senere,
+		og få full tilgang til pengene dine. Å holde denne privat er veldig viktig, siden den gir <em>hvem som helst</em> full
+        tilgang til dine moneroj! Hvis du ikke har skrevet den ned, vær så snill og gjør det!
+        <h2>Lommebokfiler gjenoprettelsespassord</h2>
+        Vær sikker på at du skriver ned dette passordet. Hvis du nullstiller enheten eller avinstallerer appen, vil
+        du trenge tilgang til lommeboka di igjen.<br/>
+        <h3>CrAzYpass</h3>
+        Hvis passordet vist her er 52 alfanunmeriske karakterer i grupper av 4 - gratulerer!
```
Comment (m2049r) [inline app/src/main/res/values-nb/help.xml]: please remove extra spaces at end of line.

**Review by m2049r — CHANGES_REQUESTED** [review]

**Comment by Codivorous** [comment]
I'm sorry for making so many commits :p

File: app/src/main/res/values-nb/strings.xml [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+
+    <string name="info_xmrto"><![CDATA[
+        <b>Du skrev inn en Bitcoin addresse.</b><br/>
+        <i>Du vil sende XMR og mottakeren vil få BTC gjennom <b>XMR.TO</b> tjenesten.</i>
+    ]]></string>
+
+    <string name="info_send_xmrto_success_order_label">XMR.TO Ordre</string>
+    <string name="info_send_xmrto_success_sending">Sender</string>
+    <string name="info_send_xmrto_success_btc">%1$s BTC</string>
+    <string name="info_send_xmrto_success_status">%1$s er %1$s (#%1$d)</string>
```
Comment (m2049r) [inline app/src/main/res/values-nb/strings.xml]: uhm... one last thing :) remove the string ```info_send_xmrto_success_status```

**Review by m2049r — CHANGES_REQUESTED** [review]

**Comment by Codivorous** [comment]
Should I just create a new pull request so you only get one commit? Because this is getting ridiculous, so unless you have some magical un-commiting tool then that's probably for the best :p

**Review by m2049r — APPROVED** [review]

**Comment by m2049r** [comment]
yay!

**Comment by m2049r** [comment]
when merging i have to option to tell github to squash commits before merging

**Comment by Codivorous** [comment]
Great! Thank you for all your assistance!

**Pull request merged**